cpu/samd5x/periph/can.c

```diff
@@ -440,19 +440,22 @@ static int _send(candev_t *candev, const struct can_frame *frame)
 
     if (frame->can_id & CAN_EFF_FLAG) {
         DEBUG_PUTS("Extended ID");
-        dev->msg_ram_conf.tx_fifo_queue[can_mm.put].TXBE_0.bit.ID = frame->can_id & CAN_EFF_MASK;
-        dev->msg_ram_conf.tx_fifo_queue[can_mm.put].TXBE_0.bit.XTD = 1;
+        dev->msg_ram_conf.tx_fifo_queue[can_mm.put].TXBE_0.reg |= CAN_TXBE_0_ID(frame->can_id & CAN_EFF_MASK)
```

Comment:
if bitfield operations are removed make them one write

assembble the whole id -> then write it